accepted/2021/platform-dependencies/platform-dependencies.md

```diff
@@ -0,0 +1,433 @@
+
+## Scenarios and User Experience
+
+### Keep other product assets up-to-date
+
+A .NET product area owner has files that need to list out the Linux packages that .NET depends on. Examples of such product areas include the following: 
+
+* Docker: A Dockerfile that [installs](https://github.com/dotnet/dotnet-docker/blob/203e3e7c2cbb6a4ac73290c603ef02d249128500/src/runtime-deps/3.1/buster-slim/amd64/Dockerfile#L7-L14) dependent packages
+* Linux packages:  A deb package that [lists](https://github.com/dotnet/installer/blob/7f8f4a894abc5f666c2bfa45ce6e7c6fe9ace71b/src/redist/targets/packaging/deb/dotnet-debian_config.json#L31-L38) the packages it depends on.
+* Docs: User documentation that [lists](https://github.com/dotnet/docs/blob/b9a09326817f9acf4dab825f62ae0a3fb2427a30/docs/core/install/linux-debian.md#dependencies) the Linux packages that are necessary to have installed for .NET.
+
+They want to ensure this list is up-to-date with .NET's requirements. They author a tool which reads from .NET's platform dependency model and compares it against their package list to ensure it aligns for a given platform and version.
+
+### Determine vulnerability exposure
+
+A security incident has been disclosed in a Linux package that could potentially affect a platform dependency that .NET has. The response team needs to have an understanding of what components of .NET have a dependency on the Linux package so that the appropriate mitigation can be implemented. They search through .NET's platform dependency model to determine which .NET components are affected and on which platforms.
+
+### Dependency-tracking tools
+
+The .NET team wants to provide users with self-service tools so that they can know, a priori, what platform dependencies their specific application has. This allows them to configure the systems that will host their application with confidence. The developers creating these tools need a single source of truth from which to derive .NET's platform dependencies. They author the tools to read from the platform dependency model to get this information.
+
+### Dependency change alerting
+
+A .NET contributor makes a change to the NativeMethods.cs file in their project. They create a PR with their changes. A bot alerts the contributor to the potential need to update .NET's platform dependency model in response to the change in the NativeMethods.cs file. The bot alerts them by adding a special label to the PR and a comment that provides instructions on next steps they should follow.
+
+### Platform EOL
+
+Debian 9 goes EOL and a maintainer of the platform dependency model needs to clean up the model to remove references to Debian 9 since those references are now obsolete. They run a CLI command to update the model automatically:
+
+```console
+dotnet-deps platform remove debian.9
+```
+
+## Requirements
+
+### Goals
+
+* Common schema capable of describing both runtime and toolchain dependencies.
```

Comment:
What do runtime and toolchain mean in this context?

Reply:
Added a definitions section which clarifies these terms: https://github.com/dotnet/designs/pull/184/commits/e38819540b74f3df2ecb087a3fb34c13fcbf4937